### PR TITLE
Remove Lead Fields from Main

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Comment by spinkelman** [comment]
QA tests pass. Waiting on approving review before merge.

**Review by xinnanlisf — APPROVED** [review]
Looks good, approved!

**Pull request merged**
